docs/guide/index.asciidoc

```diff
@@ -220,8 +220,8 @@ Then configure the elastic-apm-node module inside your service by adding the fol
 ----------------------------------
 // Add this to the VERY top of the first file loaded in your app
```

Comment:
you could also rename `app` to `service` here, but not mandatory..

Reply:
From a Node.js developers perspective, I think it's more clear to use "app" in this location. But I'm happy to reevaluate if you feel it's confusing